### PR TITLE
translation EN-RU done

### PR DESCRIPTION
**Pull request by shmudivel**
I think a few words need more context for proper translation. If possible, could you give me a link to the demo so I can see the project live?

### CODE REVIEW

**Comment by arthurits** [comment]
Hi @shmudivel,

This is a standalone program that it is compiled as a classic `.exe`. So, unfortunately, there's no live demo other than compiling and running the program.
I'll be more than glad to provide screenshots and explanations wherever you need them!

image
image

File: ErgoLux/localization/strings.ru-RU.resx [hunk 1]
```diff
@@ -402,7 +402,7 @@
     <comment/>
   </data>
   <data name="strLblPlotWindow" xml:space="preserve">
-    <value>Окно участка (секунды)</value>
+    <value>График участка (секунды)</value>
```
Comment (arthurits) [inline ErgoLux/localization/strings.ru-RU.resx:405]: Not sure about this one. This string "Plot window (seconds)" doesn't refer to a type of plot. Instead, it refers to the plots' abscissa axis range (that's the "window"). The default value is 20 seconds and, as data is aquired, the axis range changes from 0-20 to 1-21, 2-22, and so on.

This is the screenshot for this settings section:
imagen

And that's the translated one:
imagen

**Review by arthurits — COMMENTED** [review]
This is looking great @shmudivel!

All the strings starting with "strStatusTip..." contain the text shown as a tooltip whenever the user hovers over any of the texts, buttons, and icons on the status bar at the very bottom of the main window.
imagen

More screenshots:
imagen
imagen

Note to self: relocate the bottom labels in order to properly show the translated text in all languages:
imagen

**Comment by shmudivel** [comment]
@arthurits, thank you feed back. 

- "strStatusTip..." is 100% correct 
- "More screenshots: " definitely the most helpful (especially English and translated versions ) - Fixed a few typos and grammar issues
- "Not sure about this one. This string "Plot window (seconds)" - yes, you are correct. - Fixed 

There is some grammar issue: 

229426137-9c38e06a-2278-4563-a172-a4512f5e5684-2

0 hours - 0 часов 
1 hour - 1 час 
2 hours - 2 часа 
3 hours - 3 часа 
4 hours - 4 часа 
5 hours - 5 часов 

Similar problems arise with the declension of words for days, minutes, and seconds.

**Review by arthurits — APPROVED** [review]

**Comment by arthurits** [comment]
Thanks so much @shmudivel!

I'm ready to merge this PR with your translations. You did great!

I'd like to find, however, some way to handle the translation of substantives: `days`, `hours`, `minutes`, `seconds` and `milliseconds`. Unfortunately, they are stored in single-value string resources (`strFileHeader19`, `strFileHeader20`, `strFileHeader21`, `strFileHeader22`, and `strFileHeader24`) without any distinction between quantities nor sinlge/plural forms. So, I'll have to work out in the near future the best way to tackle this down for all languages.

In the meantime, would it be possible to use these alternative forms?

* Millisecond(s): N миллисекунд/а/ы
* Second(s): N секунд/а/ы
* Minute(s): N минут/а/ы
* Hour(s): N час/а/ов
* [This one seems more difficult] Day(s): N дня/ей/день

**Note to self**: be aware to also consider and implement this in [SignalAnalysis](https://github.com/arthurits/SignalAnalysis).

**Note to self**: the correct Russian translations are:

Milliseconds:
- 0 миллисекунд
- 1 миллисекунда
- 2 миллисекунды
- 3 миллисекунды
- 4 миллисекунды
- 5 миллисекунд

Seconds:
- 0 секунд
- 1 секунда
- 2 секунды
- 3 секунды
- 4 секунды
- 5 секунд

Minutes:
- 0 минут
- 1 минута
- 2 минуты
- 3 минуты
- 4 минуты
- 5 минут

Hours:
- 0 часов
- 1 час
- 2 часа
- 3 часа
- 4 часа
- 5 часов

Days:
- 0 дней
- 1 день
- 2 дня
- 3 дня
- 4 дня
- 5 дней

**Comment by shmudivel** [comment]
Thank you, glad to be helpful.

How about?:
Millisecond(s): N мc.
Second(s): N сек.
Minute(s): N мин.
Hour(s): N ч.
[This one seems more difficult] Day(s): N дн.

**Comment by arthurits** [comment]
That will certainly do, @shmudivel!

Thanks so much for your had work and your willingness to find the best translations in the given context. Phenomenal work!

**Pull request merged**